system/lib/libc-sockets.symbols

```diff
@@ -0,0 +1,23 @@
+         U __errno_location
```

Comment:
if these are for a new library, perhaps we can avoid a .symbols file, that is, to opt-in to get it the `-s` option would link it in?

Reply:
Dropped symbols file.